core/sr-primitives/src/lib.rs

```diff
@@ -64,14 +63,14 @@ pub use primitives::{TypeId, crypto::{key_types, KeyTypeId, CryptoType}};
 pub use app_crypto::RuntimeAppPublic;
 
 /// Re-export top-level arithmetic stuff.
-pub use sr_arithmetic::{
+pub use arithmetic::{
```

Comment:
So here we have the dilemma: 

always import arithmetic stuff into sr-primitives and re-export 
or refactor once and for all and make all other crates depend on sr-arithmetic. 

It should be more optimised to do the latter, right? you might need some primitive traits but not the arithmetic. There is no reason for your runtime to pull in arithmetic in this case. 
baseically we are making `sr-primitives` ,maybe too fat here. 

What do you think @thiolliere ?

Reply:
I'm happy either way.

Reply:
I'm ok either way but I agree with kian.

Reply:
Let's leave it for another PR if needed then.